schemas/com.health-union/nexus_global_context/jsonschema/2-0-0

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Health-Union/iglu-schemas/master/schemas/com.health-union/nexus_global_context/jsonschema/1-0-0",
```

Comment:
Does this version need to match the "self" block's version?

Reply:
Yeah, good catch, thanks